packages/playwright/src/plugins/gitCommitInfoPlugin.ts

```diff
@@ -67,7 +79,7 @@ const gitCommitInfoPlugin = (fullConfig: FullConfigInternal): TestRunnerPlugin =
           metadata.gitDiff = diffResult;
         }
       }
-    },
+    }
```

Comment:
I don't like unnecessary syntax changes. Please reduce newline changes / removed commans etc to a minimum.